ballerina/Package.md

```diff
@@ -8,13 +8,13 @@ sending an SMS, getting message details, making a voice call etc.
 ### Compatibility
 |                        | Version             |
 |------------------------|---------------------|
-| Ballerina Language     | Swan Lake 2201.3.0  |
+| Ballerina Language     | Swan Lake 2201.8.0  |
 | Twilio Basic API       | 2010-04-01          |     
 
 ## Report issues
-To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Extended Library repository](https://github.com/ballerina-platform/ballerina-extended-library)
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Standard Library repository](https://github.com/ballerina-platform/ballerina-library)
 
 ## Useful links
 - Discuss code changes of the Ballerina project via [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
 - Chat live with us via our [Discord server](https://discord.gg/ballerinalang).
-- Post all technical questions on Stack Overflow with the [#ballerina](https://stackoverflow.com/questions/tagged/ballerina) tag
+- Post all technical questions on Stack Overflow with the [#ballerina](https://stackoverflow.com/questions/tagged/ballerina) tag.
```

Comment:
We can just duplicate the same `Module.md` content in here.

Reply:
Addressed in [e53338e](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf)